### DISCUSSION TITLE
Missing symbols in autoimport.db

### DISCUSSION THREAD

**Discussion by rodrigoscc** [python-rope/rope#685] [General]
Hey everyone. I have been using rope with pylsp for a couple of days and noticed something odd. Autoimport suggestions contain my virtual env packages if I add the virtual env site-packages folder to the python_path preference. That's okay, but seems like rope does not scan all subfolders to generate autoimports.

For instance, I'm using `prompt_toolkit`, and the `prompt_toolkit.prompt` is returned by rope to auto import it, but the `prompt_toolkit.completion.Completer` is not. `prompt_toolkit.history.FileHistory` is listed too. So the behavior is inconsistent.

Seems to me that rope is not able to autoimport from folders inside the prompt_toolkit, since `completion` is a folder, `history` is a python file. Is this expected? or is there any configuration I can set to fix this?

I also checked the autoimport.db file and looks like the folders of prompt_toolkit are missing from there too.

**Comment by lieryan** [comment]
@rstcruzo thanks for reporting this issue. I believe this is a bug in autoimport, I've converted this discussion over to #687 and this should be fixed by #688.

Closing the discussion here, further conversations of this bug should be in #687 and/or #688.
